draft-ietf-quic-transport.md

```diff
@@ -2174,8 +2174,10 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 
 ## Path Validation Responses
 
-On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
-echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+On receiving a PATH_CHALLENGE frame, an endpoint MUST respond by echoing the
+data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.  An
+endpoint MUST NOT delay transmission of a packet containing a PATH_RESPONSE
+frame unless constrained by congestion control.
```

Comment:
Can you actually be constrained by congestion control on a new path?

Reply:
You can use PATH_CHALLENGE on existing paths as well, in which case congestion control could be a constraint.

Reply:
It's true to say that any constraint is unlikely, but not impossible.